python/pipeline/utils/eye_tracking.py

```diff
@@ -1,14 +1,10 @@
 from collections import defaultdict
-
-import matplotlib
+import matplotlib.pyplot as plt
 import pandas as pd
+import numpy as np
 
 from ..exceptions import PipelineException
 
-matplotlib.use('agg')
```

Comment:
@fabiansinz fabian, is this ok?, I deleted it because it will print a warning every time we import something from utils (because eye_tracking is imported in utils.__init__). If not, maybe I can add it back and pass keyword argument warn=False or just have it print the warning